shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -2,63 +2,93 @@ use crate::helpers::ShotoverManager;
 use test_helpers::docker_compose::DockerCompose;
 
 use crate::cassandra_int_tests::cassandra_connection;
-use cassandra_cpp::{stmt, Session};
+use cassandra_cpp::{stmt, Session, Statement, Value, ValueType};
 use serial_test::serial;
-use tracing::debug;
+
+#[derive(PartialEq, Debug)]
+enum ResultValue {
+    Text(String),
+    Varchar(String),
+    Int(i64),
+}
+
+impl ResultValue {
+    fn new(value: Value) -> ResultValue {
+        match value.get_type() {
+            ValueType::TEXT => ResultValue::Text(value.get_string().unwrap()),
+            ValueType::VARCHAR => ResultValue::Varchar(value.get_string().unwrap()),
+            ValueType::INT => ResultValue::Int(value.get_i64().unwrap()),
+            ty => todo!(
+                "The test infrastructure hasnt implemented the type {} yet, you should add it.",
+                ty
+            ),
+        }
+    }
+}
+
+fn assert_query_result(session: &Session, query: Statement, expected_rows: &[&[ResultValue]]) {
```

Comment:
I'd put this in `cassandra_int_tests/mod.rs`

Reply:
yeah, that makes sense to me